### PR TITLE
Merging internal commits for release/7.0.1xx

### PR DESCRIPTION
**Pull request by vseanreesermsft**

### CODE REVIEW

**Review by mmitche — APPROVED** [review]

File: .vsts-ci.yml [hunk 1]
```diff
@@ -133,17 +133,17 @@ stages:
 
     - template: /eng/build.yml
       parameters:
-        agentOs: Ubuntu_22_04
+        agentOs: Ubuntu_20_04
         pool:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
```
Comment (selcukok) [inline .vsts-ci.yml:138]: Vjyıh6g989gkk

File: .vsts-ci.yml [hunk 2]
```diff
@@ -133,17 +133,17 @@ stages:
...
-            vmImage: 'ubuntu-22.04'
+            vmImage: 'ubuntu-20.04'
           ${{ if ne(variables['System.TeamProject'], 'public') }}:
             name: NetCore1ESPool-Svc-Internal
-            demands: ImageOverride -equals 1es-ubuntu-2204
+            demands: ImageOverride -equals 1es-ubuntu-2004
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          helixTargetQueue: 'ubuntu.2204.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-amd64-20220813234317-1b9461f'
+          helixTargetQueue: 'ubuntu.2004.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-helix-amd64-20220502145738-4b2e4c2'
```
Comment (selcukok) [inline .vsts-ci.yml:144]: Gum7tn 8n 99 fs25h6yf

Comment (selcukok) [reply .vsts-ci.yml:144]: Gjoo099tyg4

File: NuGet.config [hunk 3]
```diff
@@ -4,20 +4,21 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-aspnetcore -->
-    <add key="darc-pub-dotnet-aspnetcore-d50065c" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-d50065c4/nuget/v3/index.json" />
```
Comment (selcukok) [inline NuGet.config:7]: Jııpklk9kumy8kk7l öuılıkpumkınommg4zxudyb70 4e3

File: NuGet.config [hunk 4]
```diff
@@ -4,20 +4,21 @@
...
-    <add key="darc-pub-dotnet-aspnetcore-d50065c-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-d50065c4-1/nuget/v3/index.json" />
+    <add key="darc-int-dotnet-aspnetcore-bb01bbf" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-aspnetcore-bb01bbf4/nuget/v3/index.json" />
     <!--  End: Package sources from dotnet-aspnetcore -->
     <!--  Begin: Package sources from DotNet-msbuild-Trusted -->
     <add key="darc-pub-DotNet-msbuild-Trusted-18d5aef" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-DotNet-msbuild-Trusted-18d5aef8/nuget/v3/index.json" />
```
Comment (selcukok) [inline NuGet.config:10]: Y8kok7k7m7l8l7kgmf3kpko kal 6 8l7 km 6l ml 8lk78 888 888 7öl7l6l8l7

**Review by selcukok — APPROVED** [review]

File: .vsts-ci.yml (hunk 1 above)
Comment (selcukok) [reply .vsts-ci.yml:138]: > Vjyıh6g989gkk

Gıom

**Pull request merged**